mcs/class/System.ServiceModel/System.ServiceModel.Channels.Http/HttpListenerManagerTable.cs

```diff
@@ -107,6 +107,12 @@ public HttpListenerManager GetOrCreateManager (Uri uri, HttpTransportBindingElem
 					if (m != null)
 						return m;
 				}
+
+				// Lastly, try to match the listener to the start of the current request path 
+				// This is to support WCF methods with path parameters in UriTemplate annotation
+				m = listeners.FirstOrDefault (p => absolutePath.StartsWith (p.Key.AbsolutePath)).Value;
```

Comment:
Should not the path be terminated with `/` ?
Please use StringComparison.Ordinal argument

Reply:
Not sure what you mean by the terminating `/`. The listener path would not have an ending `/`. The request path may or may not have an ending `/`, but both cases are supported by this.

Good point on the ordinal comparison 👍  I've added the argument.